Makefile

```diff
@@ -29,15 +29,21 @@ postman:
 	# and if needed a token.
 	# Use env=local to bring up a local version of the app and test against it
 	# For example: make postman env=test token=<MY_TOKEN>
-	$(eval BCDA_SSAS_CLIENT_ID=$(shell docker exec bcda-app_api_1 env | grep BCDA_SSAS_CLIENT_ID | cut -d'=' -f2))
-	$(eval BLACKLIST_CLIENT_ID=$(shell docker exec bcda-app_api_1 env | grep BLACKLIST_CLIENT_ID | cut -d'=' -f2))
-	$(eval BLACKLIST_CLIENT_SECRET=$(shell docker exec bcda-app_api_1 env | grep BLACKLIST_CLIENT_SECRET | cut -d'=' -f2))
-	$(eval BCDA_SSAS_SECRET=$(shell docker exec bcda-app_api_1 env | grep BCDA_SSAS_SECRET | cut -d'=' -f2))
+	$(eval BLACKLIST_CLIENT_ID=$(shell docker-compose exec api env | grep BLACKLIST_CLIENT_ID | cut -d'=' -f2))
+	$(eval BLACKLIST_CLIENT_SECRET=$(shell docker-compose exec api env | grep BLACKLIST_CLIENT_SECRET | cut -d'=' -f2))
+
+	# Set up valid client credentials
+	$(eval ACO_CMS_ID = A9994)
+	$(eval CLIENT_TEMP := $(shell docker-compose exec api sh -c 'bcda reset-client-credentials --cms-id $(ACO_CMS_ID)'|tail -n2))
+	$(eval CLIENT_ID:=$(shell echo $(CLIENT_TEMP) |awk '{print $$1}'))
+	$(eval CLIENT_SECRET:=$(shell echo $(CLIENT_TEMP) |awk '{print $$2}'))
+
 	docker-compose -f docker-compose.test.yml build postman_test
 	@docker-compose -f docker-compose.test.yml run --rm postman_test test/postman_test/BCDA_Tests_Sequential.postman_collection.json \
-	-e test/postman_test/$(env).postman_environment.json --global-var "token=$(token)" --global-var clientId=$(BCDA_SSAS_CLIENT_ID) --global-var clientSecret=$(BCDA_SSAS_SECRET) \
+	-e test/postman_test/$(env).postman_environment.json --global-var "token=$(token)" --global-var clientId=$(CLIENT_ID) --global-var clientSecret=$(CLIENT_SECRET) \
```

Comment:
The clientID in this case can be regenerated ([see](https://github.com/CMSgov/bcda-app/pull/680/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R36-R40)). 

Since we updated the BCDA_SSAS_CLIENT_ID to be the admin credentials (see the change in local.env), we need to ensure that a valid set of credentials are used for this ACO. Since each E2E test (postman + smoke) reset/generate credentials prior to using them, we do not risk in using invalid creds.